docs/pages/versions/unversioned/sdk/speech.md

```diff
@@ -46,3 +46,20 @@ Determine whether the Text-to-speech utility is currently speaking. Will return
 #### Returns
 
 Returns a Promise that resolves to a boolean, `true` if speaking, `false` if not.
+
+### `Expo.Speech.getAvailableVoicesAsync()` (iOS only)
+
+Returns list of all available voices.
+
+#### Returns
+
+List of `Voice` objects.
+
+##### `Voice`
+
+|    Field   |               Type               |
+|:----------:|:--------------------------------:|
+| identifier |              string              |
+|    name    |              string              |
+|   quality  | string ("Default" or "Enhanced") |
```

Comment:
What do you think about enum `Speech.VoiceQuality`? I'd prefer that our users use enums instead of plain strings as there is higher probability they do a typo in a plain string (IDE suggests enums) and also we wouldn't be able to change such values in the future, without a breaking change.

```typescript
enum VoiceQuality {
  Default = "default",
  Enhanced = "enhanced",
}
```